modules/ding_webtrekk/ding_webtrekk.admin.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @file
+ * Administration form definition.
+ */
+
+/**
+ * Define admin page.
+ */
+function ding_webtrekk_admin_settings_form() {
+  $form = [];
+  
+  $form['webtrekk_ti_domain'] = [
+    '#type' => 'textfield',
+    '#title' => t('TagIntegration Domain'),
+    '#description' => t('Enter the TagIntegration domain here, if the TagIntegration file should be loaded from aWebtrekk server.'),
+    '#default_value' => variable_get('webtrekk_ti_domain', ''),
+  ];
+  $form['webtrekk_ti_id'] = [
+    '#type' => 'textfield',
+    '#title' => t('TagIntegration ID'),
+    '#description' => t('Enter your TagIntegration customer ID here, if the TagIntegration file should be loaded from aWebtrekk server.'),
```

Comment:
The final part of the description "if the TagIntegration file should be loaded from aWebtrekk server" seems odd. The administration interface provides no other way to provide a TagIntegration file.